sprint_misison_7/queries.sql

```diff
@@ -0,0 +1,153 @@
+/*
+  다음 경우들에 대해 총 14개의 SQL 쿼리를 작성해 주세요.
+  예시로 값이 필요한 경우 적당한 값으로 채워넣어서 작성하면 됩니다. 
+*/
+
+/*
+  1. 내 정보 업데이트 하기
+  - 닉네임을 "test"로 업데이트
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+UPDATE users
+SET nickname = 'test'
+WHERE id = 1;
+
+
+/*
+  2. 내가 생성한 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/ 
+SELECT *
+FROM products
+WHERE seller_id = 1
+ORDER BY created_at DESC
```

Comment:
말씀드렸다시피 정렬이 생각보다 정합성이 중요해요.
시간 값을 하나만 정렬에 사용하면, 다양한 애플리케이션 문제가 발생할 수 있어요.

이 문제는 별거아닌 것 처럼 보이지만, 일관된 정렬을 보장하지 않아서, 페이징 중복/누락 등이 발생할 수 있습니다.

```suggestion
ORDER BY created_at DESC, id DESC
```

Reply:
넵 확인했습니다